### PR TITLE
Av2 846 preference desktop new design

### PR DESCRIPTION
**Pull request by ronakamani**
**Story/Ticket Reference ID from JIRA:**
https://intersective.atlassian.net/browse/AV2-846

**Additional Comments:**
`Add here`

**Checklist:**

- [ x ] Unit test completed?: (Y/N)
- [ x ] Docs folder up to date?: (Y/N)
- [ x ] Add if anything missed: (Y/N)

All Checklist are okay.

This PR looks great - it's ready to merge! Please review and let's ship it. :)

### CODE REVIEW

File: src/app/preference/preference-update/preference-update.component.ts [hunk 1]
```diff
@@ -76,20 +102,28 @@ export class PreferenceUpdateComponent implements OnInit, OnDestroy {
     return result;
   }
 
+  private getKey() {
```
Comment (trtshen) [inline src/app/preference/preference-update/preference-update.component.ts:109]: @ronakamani can you please add docblock for this getkey() function?

File: src/app/preference/preference-update/preference-update.component.ts [hunk 2]
```diff
@@ -111,15 +145,11 @@ export class PreferenceUpdateComponent implements OnInit, OnDestroy {
     }
     return;
   }
-
-  /**
-   * @name back
-   * @description manual back button to go back to a pre-structured routing
-   *              (back to "/preference")
-   */
```
Comment (trtshen) [inline src/app/preference/preference-update/preference-update.component.ts]: @ronakamani the docblock got deleted, can you please add back?

File: src/app/tabs/tabs-routing.module.ts [hunk 3]
```diff
@@ -36,6 +36,15 @@ const routes: Routes = [
           }
         ]
       },
+      {
+        path: 'preference',
```
Comment (trtshen) [inline src/app/tabs/tabs-routing.module.ts:40]: @ronakamani do we need preference in tab?

**Review by trtshen — CHANGES_REQUESTED** [review]

File: src/app/tabs/tabs-routing.module.ts (hunk 3 above)
Comment (ronakamani) [reply src/app/tabs/tabs-routing.module.ts:40]: @Chaw, yes we need that, if we delete this routing in the tab routing it won't show tabs toolbar on top of preference page.

**Comment by ronakamani** [comment]
the final branch is merged.

**Pull request closed without merging**